src/System.CommandLine/Option.cs

```diff
@@ -58,15 +46,12 @@ public abstract class Option : IdentifierSymbol, IValueDescriptor
             {
                 AddAlias(aliases[i]);
             }
-
-            argument.AddParent(this);
-            _argument = argument;
         }
 
         /// <summary>
         /// Gets the <see cref="Argument">argument</see> for the option.
         /// </summary>
-        internal virtual Argument Argument => _argument;
+        internal abstract Argument Argument { get; }
```

Comment:
This will break custom implementations of `Option`, we don't have any in S.CL source code, expect for Option<T>, but there may be users that do have them. 

Is that not an issue?

Reply:
@Jozkee this method is `internal` so it won't be an issue